packages/core/crates/core-hopr/README.md

```diff
@@ -1,3 +1,15 @@
 # core-hopr
 
 Implements the main public interface for the core HOPR library - that is the `Hopr` type.
+
+Tha application representing the entire core functionality, including:
+- p2p
+- transport
+- packet construction
+- mixing
+- strategies
+- crypto
+
+## Detail
+
+The package defines a core application definition.
```

Comment:
I'd maybe add more high-level overview here about how the adaptors work and how that plays together with the swarm.

Reply:
OK, I added a short rationale.